### DISCUSSION TITLE
Remove Self-hosted runner config from linux machine

### DISCUSSION THREAD

**Discussion by zeeshan-812** [community/community#37968] [Actions]
Hi,

I accidentally removed a Runner from my repo settings and did not save the `runner remove token`. I want to add a runner in the same linux machine but I get the following error:

```
Cannot configure the runner because it is already configured. To reconfigure the runner, run 'config.cmd remove' or './config.sh remove' first.
```

Is there a way I can remove the previous config and add the new one on the same machine without the runner remove token?

**Comment by mickeygousset** [comment]
If the runner folder is still on your linux machine, you should be able to go into that folder and run the ./config.sh remove and it will remove the files. Is that not working for you?
